benchmark/buffers/buffer-hex.js

```diff
@@ -3,8 +3,8 @@
 const common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
-  len: [0, 1, 64, 1024],
-  n: [1e7]
+  len: [64, 1024],
+  n: [1e6]
```

Comment:
The same as above: super tiny buffers just do not really matter performance wise in real applications and having benchmarks that show any "significant" difference for such entries would not have any actual influence.

The iteration count was reduced to a more reasonable number.